build.sh

```diff
@@ -2,22 +2,31 @@
 set -ex
 
 export GITHUB_USERNAME=$1
-export GITHUB_PASSWORD=$2
+export GITHUB_TOKEN=$2
 export AWS_ACCESS_KEY_ID=$3
 export AWS_ACCESS_KEY=$4
 export REPO=$5
 export BRANCH=$6
-export CORE_TAG_NAME="5.0.dev1"
 export CORE_BRANCH="master"
 
 set +e
 . /etc/profile.d/rvm.sh
 set -e
 
-curl -u $GITHUB_USERNAME:$GITHUB_PASSWORD https://raw.githubusercontent.com/cloudify-cosmo/${REPO}/${CORE_BRANCH}/packages-urls/common_build_env.sh -o ./common_build_env.sh &&
-source common_build_env.sh &&
-curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-common/${CORE_BRANCH}/packaging/common/provision.sh -o ./common-provision.sh &&
-source common-provision.sh
+set +x
+export current_branch=$CORE_BRANCH && curl -u $GITHUB_USERNAME:$GITHUB_TOKEN -fO "https://raw.githubusercontent.com/cloudify-cosmo/${REPO}/${CORE_BRANCH}/packages-urls/common_build_env.sh" || \
```

Comment:
(same below)
this logic with `export && curl || export && curl` is weird, maybe rewrite as an if, or maybe add `{}` or maybe just add better indentation/spacing/newlines so that this is easier to follow

Reply:
(also same in the 2nd file)
also note that in line 10 we have a hardcoded `export CORE_BRANCH="master"` which is not overridden anyway